cfgov/jinja2/v1/_includes/organisms/carousel.html

```diff
@@ -9,35 +9,34 @@
    Creates markup for a Carousel organism, which displays a set of
    blocks of content that can be navigated through sequentially.
 
-   value._items:                Carousel content panels.
+   value:                       List of carousel content panels.
 
-   item.title:                  Panel title.
+   item.heading:                Panel heading.
 
-   item.body:                   Panel body text.
+   item.thumbnail_heading:      Optional. Shortened version of panel heading.
 
-   item.link.text:              Panel link text.
+   item.text:                   Panel body text.
 
-   item.link.url:               Panel link URL.
+   item.link_text:              Panel link text.
 
-   item.image.url:              URL to panel image.
+   item.link_url:               Panel link URL.
 
-   item.image.alt_text:         Alt text for panel image.
-
-   item.image.thumbnail_url:    URL to panel thumbnail image.
+   item.image:                  Panel Wagtail image.
 
    ========================================================================== #}
 
 {% macro render_carousel_item( item ) %}
 <section class="o-carousel_item">
     <div class="o-carousel_item-text">
-        <h2>{{ item.title }}</h2>
-        <p>{{ item.body }}</p>
-        <p><a href="{{ item.link.url }}">{{ item.link.text }}</a></p>
+        <h2>{{ item.heading }}</h2>
+        <p>{{ item.text }}</p>
+        <p><a href="{{ item.link_url }}">{{ item.link_text }}</a></p>
     </div>
     <div class="o-carousel_item-visual">
         <img class="o-carousel_item-img"
-             src="{{ item.image.url }}"
-             alt="{{ item.image.alt_text }}"
+             {% set original_image = image( item.image, 'original' ) %}
```

Comment:
I'd suggest moving this up one line, to not break up the `<img>` tag. (Below on the thumbnail, too.)